changelogs/unreleased/gh-9979-space-format-object.md

```diff
@@ -0,0 +1,3 @@
+## feature/box
+
+* Populated `box.space` objects with the new field `format_object` (gh-9979).
```

Comment:
You've just populated `space` objects of `net.box` with `format_object` as well. Please, write about them in your:
1. Commit message.
2. Doc request.
3. Changelog entry.